framework/console/controllers/BaseMigrateController.php

```diff
@@ -465,11 +493,35 @@ public function actionCreate($name)
             throw new Exception("The migration name should contain letters, digits and/or underscore characters only.");
         }
 
-        $name = 'm' . gmdate('ymd_His') . '_' . $name;
-        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $name . '.php';
+        $className = 'm' . gmdate('ymd_His') . '_' . $name;
+        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $className . '.php';
 
         if ($this->confirm("Create new migration '$file'?")) {
-            $content = $this->renderFile(Yii::getAlias($this->templateFile), ['className' => $name]);
+            if (preg_match('/^Create(.+)$/', $name, $matches)) {
```

Comment:
Laravel generators use underscore syntax and ruby on rails use camelcase.

@cebe @klimov-paul which do you prefer syntax?